app/views/hosts/show.html.erb

```diff
@@ -1,5 +1,11 @@
 <% javascript 'charts', 'hosts' %>
-<% title @host.to_label, text: @host.to_label, icon: icon(@host.operatingsystem, path: true) %>
+<% title @host.to_label,
+  text: @host.to_label,
+  icon: {
+    alt: label(@host.operatingsystem),
```

Comment:
instead of `to_label` and this `label` wrapper, try to use `title`, it is a field that calculated from `to_label` and needs to be presence.

Reply:
I will still need to use the `label` wrapper since `@host.operatingsystem` is nil sometimes.
Does it still give value to use `title`?

Reply:
@amirfefer any additional concerns?